docs/installation/install-client.rst

```diff
@@ -119,38 +119,29 @@ Whenever you need to run Flocker CLI commands, ensure you are in the virtualenv:
 Installing on OS X
 ==================
 
-Install the `Homebrew`_ package manager.
-
-Make sure Homebrew has no issues:
-
-.. prompt:: bash alice@mercury:~$
-
-   brew doctor
-
-Fix anything which ``brew doctor`` recommends that you fix by following the instructions it outputs.
-
-If you have a previous version of Flocker tapped, you can run the following to remove it:
+Install ``pip`` and ``virtualenv`` on your machine at the system level:
 
 .. prompt:: bash alice@mercury:~$
 
-   brew uninstall flocker-<old version>
+   sudo python -m ensurepip
+   sudo pip install virtualenv
 
-Add the ``ClusterHQ/tap`` tap to Homebrew and install ``flocker``:
+To install ``flocker-cli`` in a Python virtualenv, run:
```

Comment:
Yea, just following existing practice for now. Acknowledged about maybe we should change this... although flocker-cli is future resistant :), in that it will always be accurate even if we change what is in that package. Mostly just not changing because not changing is easier than changing.